### PR TITLE
Extracted method for parsing left group char in `TexFormulaParser`

### PR DESCRIPTION
**Pull request by Lehonti**
I see that this parser is mutation-based...In the future I'd like to make it more immutable, more functional, and more generation-based. This is a step in that direction (although I don't think that's obvious right now). Thankfully all methods are private so refactoring is easier

### CODE REVIEW

**Comment by ForNeVeR** [comment]
> I see that this parser is mutation-based...In the future I'd like to make it more immutable, more functional, and more generation-based.

We even have an issue about that: #29

File: src/XamlMath.Shared/TexFormulaParser.cs [hunk 1]
```diff
@@ -296,6 +293,20 @@ internal TexFormula Parse(SourceSpan value, string? textStyle, ICommandEnvironme
             return formula;
         }
 
+        private void ProcessLeftGroupChar(SourceSpan value, ref int position, string textStyle, ICommandEnvironment environment, TexFormula formula, int initialPosition)
```
Comment (ForNeVeR) [inline src/XamlMath.Shared/TexFormulaParser.cs:296]:
```suggestion
        private void ProcessLeftGroupChar(SourceSpan value, ref int position, string? textStyle, ICommandEnvironment environment, TexFormula formula, int initialPosition)
```

Remember the compiler warnings and that `textStyle` is nullable.

**Review by ForNeVeR — CHANGES_REQUESTED** [review]
Other than a minor issue, LGTM!

**Comment by Lehonti** [comment]
No problem, I accepted your suggestions :)

**Review by ForNeVeR — APPROVED** [review]
Thanks!

**Pull request merged**